src/Symfony/Component/PropertyAccess/Exception/UnexpectedTypeException.php

```diff
@@ -18,8 +18,4 @@
  */
 class UnexpectedTypeException extends RuntimeException
 {
-    public function __construct($value, $expectedType)
```

Comment:
That would work, I'll make the changes. Good point that the `PropertyPath` exception is not meant to be caught anyway.